File: GTTS/INL.VettingService.Models/PersonVetting_Item.cs
using System;
using System.Collections.Generic;

namespace INL.VettingService.Models
{
    public class PersonVetting_Item : IPersonVetting_Item
    {
        public long PersonsVettingID { get; set; }
        public long PersonsUnitLibraryInfoID { get; set; }
        public long PersonID { get; set; }
        public string GivenName { get; set; }
        public string MiddleName1 { get; set; }
        public string MiddleName2 { get; set; }
        public string MiddleName3 { get; set; }
        public string FamilyName { get; set; }
        public string Name1 { get; set; }
        public string Name2 { get; set; }
        public string Name3 { get; set; }
        public string Name4 { get; set; }
        public string Name5 { get; set; }
        public string FirstMiddleNames { get; set; }
        public string LastNames { get; set; }
        public string POBCountryINKCode { get; set; }
        public string POBStateINKCode { get; set; }
        public DateTime? DOB { get; set; }
        public char Gender { get; set; }
        public string NationalID { get; set; }
        public int? POBCityID { get; set; }
        public string POBCityName { get; set; }
        public int? POBStateID { get; set; }
        public string POBStateName { get; set; }
        public int? POBCountryID { get; set; }
        public string POBCountryName { get; set; }
        public long? UnitID { get; set; }
        public string UnitName { get; set; }
        public string UnitGenID { get; set; }
        public string UnitAgencyName { get; set; }
        public string UnitParents { get; set; }
        public string UnitParentsEnglish { get; set; }
        public string UnitBreakdownLocalLang { get; set; }
        public string UnitAcronym { get; set; }
        public string JobTitle { get; set; }
        public string RankName { get; set; }
        public long VettingBatchID { get; set; }
        public int VettingPersonStatusID { get; set; }
        public string VettingStatus { get; set; }
        public DateTime? VettingStatusDate { get; set; }
        public string VettingNotes { get; set; }
        public DateTime? ClearedDate { get; set; }
        public int? AppUserIDCleared { get; set; }
        public DateTime? DeniedDate { get; set; }
        public int? AppUserIDDenied { get; set; }
        public int ModifiedByAppUserID { get; set; }
        public DateTime ModifiedDate { get; set; }
        public int? LastVettingStatusID { get; set; }
        public string LastVettingStatusCode { get; set; }
        public DateTime? LastVettingStatusDate { get; set; }
        public byte? LastVettingTypeID { get; set; }
        public string LastVettingTypeCode { get; set; }
        public long ParticipantID { get; set; }
        public string ParticipantType { get; set; }

        public bool IsRemoved { get; set; }
        public bool RemovedFromEvent { get; set; }
        public bool RemovedFromVetting { get; set; }
        public string LeahyHitResultCode { get; set; }
        public string VettingActivityType { get; set; }
        public bool IsReVetting { get; set; }
    }
}

File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.VettingService.Models/person-vetting_item.ts


import { IPersonVetting_Item } from './iperson-vetting_item';

export class PersonVetting_Item implements IPersonVetting_Item {
  
	public PersonsVettingID: number = 0;
	public PersonsUnitLibraryInfoID: number = 0;
	public PersonID: number = 0;
	public GivenName: string = "";
	public MiddleName1: string = "";
	public MiddleName2: string = "";
	public MiddleName3: string = "";
	public FamilyName: string = "";
	public Name1: string = "";
	public Name2: string = "";
	public Name3: string = "";
	public Name4: string = "";
	public Name5: string = "";
	public FirstMiddleNames: string = "";
	public LastNames: string = "";
	public POBCountryINKCode: string = "";
	public POBStateINKCode: string = "";
	public DOB?: Date;
	public Gender: string = "";
	public NationalID: string = "";
	public POBCityID?: number;
	public POBCityName: string = "";
	public POBStateID?: number;
	public POBStateName: string = "";
	public POBCountryID?: number;
	public POBCountryName: string = "";
	public UnitID?: number;
	public UnitName: string = "";
	public UnitGenID: string = "";
	public UnitAgencyName: string = "";
	public UnitParents: string = "";
	public UnitParentsEnglish: string = "";
	public UnitBreakdownLocalLang: string = "";
	public UnitAcronym: string = "";
	public JobTitle: string = "";
	public RankName: string = "";
	public VettingBatchID: number = 0;
	public VettingPersonStatusID: number = 0;
	public VettingStatus: string = "";
	public VettingStatusDate?: Date;
	public VettingNotes: string = "";
	public ClearedDate?: Date;
	public AppUserIDCleared?: number;
	public DeniedDate?: Date;
	public AppUserIDDenied?: number;
	public ModifiedByAppUserID: number = 0;
	public ModifiedDate: Date = new Date(0);
	public LastVettingStatusID?: number;
	public LastVettingStatusCode: string = "";
	public LastVettingStatusDate?: Date;
	public LastVettingTypeID?: number;
	public LastVettingTypeCode: string = "";
	public ParticipantID: number = 0;
	public ParticipantType: string = "";
	public IsRemoved: boolean = false;
	public RemovedFromEvent: boolean = false;
	public RemovedFromVetting: boolean = false;
	public LeahyHitResultCode: string = "";
	public VettingActivityType: string = "";
	public IsReVetting: boolean = false;
  
}


